libraries/Update/src/HttpsOTAUpdate.cpp

```diff
@@ -1,11 +1,9 @@
-/* OTA task
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
 
-   This example code is in the Public Domain (or CC0 licensed, at your option.)
-
-   Unless required by applicable law or agreed to in writing, this
-   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
-   CONDITIONS OF ANY KIND, either express or implied.
-*/
```

Comment:
revert this change please.

Reply:
@me-no-dev - reverted.
HttpsOTAUpdate.cpp/.h are marked as CC0
Update.h/Updater.cpp are marked as Apache-2.0

Is it correct?

Reply:
I wrote Updater, not HttpsOTAUpdate, so the user who wrote it, put that license there :)